src/components/Order.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+function Order(props) {
+  const [orders, setOrders] = useState(0);
+
+  function orderOne() {
+    return setOrders(orders + 1);
+  }
```

Comment:
State and `onClick` function work very well together - well done! This is a really nice and clean component👏